spec/lib/discourse_js_processor_spec.rb

```diff
@@ -92,7 +92,7 @@ class MyClass {
     JS
 
     result = DiscourseJsProcessor.transpile(script, "blah", "blah/mymodule")
-    expect(result).to include("_applyDecoratedDescriptor")
+    expect(result).to include("static #_ = dt7948.n")
```

Comment:
Is `dt7948` name stable?

Reply:
[Yup!](https://github.com/ef4/decorator-transforms/blob/1e148d2d0fdb54b4c90530c1c783aaa7bc28c181/src/global-id.ts#L1-L3)

(obviously wouldn't consider it 'public API', but I think it's stable enough for us to use in this test assertion)